ui/app/AppLayouts/Browser/+noWebEngine/BrowserLayout.qml

```diff
@@ -0,0 +1,22 @@
+import QtQuick
+
+import AppLayouts.Browser.stores as BrowserStores
+
+// dummy container/section for mobile
+Item {
```

Comment:
Still crashing here. I'd just make it a `StatusSectionLayout`. This dummy container could become a permanent resident in the end..

Reply:
Done